### PR TITLE
Update JDKs in .travis.yml

### PR DESCRIPTION
**Pull request by lorenzleutgeb**

### CODE REVIEW

**Comment by lorenzleutgeb** [comment]
This build is failing because Gradle 4.4 does not support Java 11. Consider merging #178 first, which should make this PR build fine.

File: .travis.yml [hunk 1]
```diff
@@ -5,11 +5,25 @@ sudo: true
...
+# We test against Java releases:
+#  - greater than 7
+#  AND
+#    - with LTS and in "Premier Support" phase
+#    OR
+#    - not succeeded by any other release
+# https://www.oracle.com/technetwork/java/java-se-support-roadmap.html
 # https://docs.travis-ci.com/user/languages/java#Testing-Against-Multiple-JDKs
 jdk:
+  # LTS and in "Permier Support" as of 2019-03 (until 2022-03)
```
Comment (rtaupe) [inline .travis.yml]: I think this should be "Premier Support".

File: .travis.yml [hunk 2]
```diff
@@ -5,11 +5,25 @@ sudo: true
...
   - openjdk8
   - oraclejdk8
-  - oraclejdk9
+  # LTS and in "Permier Support" as of 2019-03 (until 2023-09)
```
Comment (rtaupe) [inline .travis.yml]: I think this should be "Premier Support".

**Comment by rtaupe** [comment]
This build now fails because #175 has not yet been merged.

**Comment by lorenzleutgeb** [comment]
I merged #175 and am now going forward by merging this PR without further review by @rtaupe.

**Pull request merged**